src/config/styles.js

```diff
@@ -4,7 +4,11 @@ export const reactBlue = (opacity = 1) => `rgba(111, 207, 240, ${opacity})` // #
 
 export const blue1 = (opacity = 1) => `rgba(0,41,56, ${opacity})` // #002938
 
-export const blue2 = (opacity = 1) => `rgba(87,178,212,${opacity})`
+export const blue2 = (opacity = 1) => `rgba(87,178,212,${opacity})` 
+
+export const REACTBLUEDARK = '#57B2D4'
```

Comment:
maybe `REACT_BLUE_DARK`? its more readable? :)

Reply:
:+1: I'll update this